README.rst

```diff
@@ -27,21 +27,23 @@ PennyLane Forest Plugin
 
 .. header-start-inclusion-marker-do-not-remove
 
-Contains the PennyLane Forest plugin. This plugin allows three Rigetti devices to work with
+Contains the PennyLane Forest plugin. This plugin allows different Rigetti-pyQuil devices to work with
```

Comment:
There seeem to be many terms used for Rigetti software/devices:

Forest:
> PyQuil, along with quilc, the QVM, and other libraries, make up what is called the Forest SDK.

PyQuil:
> PyQuil is a Python library for quantum programming using Quil, the quantum instruction language developed at Rigetti Computing.

Maybe just good to leave this as is?
```suggestion
Contains the PennyLane Forest plugin. This plugin allows different Rigetti devices to work with
```